Sofa/framework/Core/src/sofa/core/DevBaseMonitor.h

```diff
@@ -35,17 +36,18 @@ namespace sofa::core
  *
  *
  */
-class DevBaseMonitor : public virtual objectmodel::BaseObject
-{
-public:
-    SOFA_CLASS(DevBaseMonitor, objectmodel::BaseObject);
-    SOFA_BASE_CAST_IMPLEMENTATION(DevBaseMonitor)
-
-    /// Destructor
-    ~DevBaseMonitor() override {}
-    /// Compute metrics
-    virtual void eval() = 0;
-};
+// SOFA_ATTRIBUTE_DEPRECATED__DEVBASEMONITOR()
+// class DevBaseMonitor : public virtual objectmodel::BaseObject
+// {
+// public:
+//     SOFA_CLASS(DevBaseMonitor, objectmodel::BaseObject);
+//     SOFA_BASE_CAST_IMPLEMENTATION(DevBaseMonitor)
+//
+//     /// Destructor
+//     ~DevBaseMonitor() override {}
+//     /// Compute metrics
+//     virtual void eval() = 0;
+// };
 
```

Comment:
If you comment everything, then it is more of a deletion than a depreciation.

Reply:
![image](https://github.com/user-attachments/assets/f479660f-bd48-44ce-a445-86da52b35ed6)